interleave-in-parent.md

```diff
@@ -0,0 +1,172 @@
+- If the amount of interleaved data stored for any Primary Key value of the root table is larger than [a range's maximum size](configure-replication-zones.html#replicaton-zone-format) (64MB by default), the interleaved optimizations will be diminished.<br/><br/>For example, if one customer has 200MB of order data, their data is likely to be spread across multiple key-value ranges and CockroachDB will not be able to access it as quickly, despite it being interleaved.
+
+## Syntax
+
+{% include sql/diagrams/interleave.html %}
+
+## Parameters
+
+| Parameter | Description |
+|-----------|-------------|
+| `CREATE TABLE ...` | For help with this section of the syntax, [`CREATE TABLE`](create-table.html).
+| `parent_table` | The name of the parent table you want to interleave the new child table into |
+| `child_columns` | A comma-separated list of columns from the child table's Primary Key that represent the interleave prefix |
+
+## Requirements
+
+- You can only interleave tables when creating the child table.
+- Each child table's Primary Key must contain its parent table's Primary Key as a prefix (known as the **interleave prefix**).<br/><br/>For example, if the parent table's primary key is `(a INT, b STRING)`, the child table's primary key could be `(a INT, b STRING, c DECIMAL)`.
+  {{site.data.alerts.callout_info}}This requirement is enforced only by ensuring that the columns use the same data types. However, we recommend ensuring the columns refer to the same data element by using the  <a href="constraints.html#foreign-keys">Foreign Key constraint</a>.{{site.data.alerts.end}}
+- Interleaved tables cannot be the child of more than 1 parent table. However, each parent table can have many children tables. Children tables can also be parents of interleaved tables.
+
+## Recommendations
+
+- Use interleaved tables when your schema forms a hierarchy, and the Primary Key of the root table (for example, a "user ID" or "account ID") is a parameter to most of your queries.
+- To enforce the relationship between the parent and children table's Primary Keys, use [Foreign Key constraints](constraints.html#foreign-keys) on the child table.
+- In cases where you're uncertain if interleaving tables will improve your queries' performance, test how tables perform under load when they're interleaved and when they aren't.
+
+## Examples
+
+### Interleaving Tables
+
+This example creates an interleaved hierarchy between `customers`, `orders`, and `packages`, as well as the appropriate Foreign Key constraints. You can see that each child table uses its parent table's Primary Key as a prefix of its own Primary Key (the **interleave prefix**).
+
+~~~ sql
+> CREATE TABLE customers (
+    id INT PRIMARY KEY,
+    name STRING
+  );
+
+> CREATE TABLE orders (
+    customer INT, 
+    id INT,
+    total DECIMAL, 
+    PRIMARY KEY (customer, int), 
+    CONSTRAINT fk_customer FOREIGN KEY (customer) REFERENCES customers
+    ) INTERLEAVE IN PARENT customers (customer)
```

Comment:
Having `INTERLEAVE...` on the same line as `CONSTRAINT` feels a bit weird to me. Not sure about a better way to format, though...